paymentsheet/src/main/java/com/stripe/android/link/LinkControllerInteractor.kt

```diff
@@ -339,11 +339,11 @@ internal class LinkControllerInteractor @Inject constructor(
             )
     }
 
-    suspend fun createPaymentMethod(): LinkController.CreatePaymentMethodResult {
-        val paymentMethodResult = performCreatePaymentMethod()
+    suspend fun createPaymentMethod(apiKey: String? = null): LinkController.CreatePaymentMethodResult {
```

Comment:
👀 `apiKey` needed to create the PM for the MoR. See http://go/lng/onramp-requests-auth